### PR TITLE
fix: [PL-42860]: Added the correct string to be displayed for User Invite

### PR DESCRIPTION
**Pull request by adiyaar24**
## Describe your changes
`Currently it still shows ‘USER_INVITED_SUCCESSFULLY’ although the user automatically gets added to the list of users in the Harness platform. The status from BE should come as USER_ADDED_SUCCESSFULLY as no more action is required from invited/inviting user, thanks`

So the above issue has been fixed in this PR

## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- 
  Specific builds
  - `trigger manager`
  - `trigger dms`
  - `trigger ng_manager`
  - `trigger cvng `
  - `trigger cmdlib`
  - `trigger template_svc`
  - `trigger events_fmwrk_monitor`
  - `trigger event_server`
  - `trigger change_data_capture`
  -  Trigger multiple builds together: For eg: `trigger dms, manager`
- Immutable delegate `trigger publish-delegate`
</details>

## [Latest PR Check Triggers](https://github.com/harness/harness-core/blob/develop/.github/pull_request_template.md)

<details>
  <summary>PR Check triggers</summary>
You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ut0, ut1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
  - CodeFormat: `trigger codeformat`
  - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- Trigger CommonChecks: `trigger commonchecks`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- UnitTests-ALL: `trigger uts`
- UnitTests-0: `trigger ut0`
- UnitTests-1: `trigger ut1`
- UnitTests-2: `trigger ut2`
- UnitTests-3: `trigger ut3`
- UnitTests-4: `trigger ut4`
- UnitTests-5: `trigger ut5`
- UnitTests-6: `trigger ut6`
- UnitTests-7: `trigger ut7`
- UnitTests-8: `trigger ut8`
- UnitTests-9: `trigger ut9`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
- Go Build: `trigger gobuild`
- Validate_Reviews: `trigger review`
- Module Dependency Check: `trigger mdc`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/55797)

### CODE REVIEW

File: 120-ng-manager/src/test/java/io/harness/ng/core/invites/api/InviteServiceImplTest.java [hunk 1]
```diff
@@ -12,13 +12,7 @@
...
 import static io.harness.exception.WingsException.USER;
 import static io.harness.ng.core.invites.InviteType.ADMIN_INITIATED_INVITE;
-import static io.harness.ng.core.invites.dto.InviteOperationResponse.ACCOUNT_INVITE_ACCEPTED;
-import static io.harness.ng.core.invites.dto.InviteOperationResponse.INVITE_EXPIRED;
-import static io.harness.ng.core.invites.dto.InviteOperationResponse.INVITE_INVALID;
-import static io.harness.ng.core.invites.dto.InviteOperationResponse.USER_ALREADY_ADDED;
-import static io.harness.ng.core.invites.dto.InviteOperationResponse.USER_ALREADY_INVITED;
-import static io.harness.ng.core.invites.dto.InviteOperationResponse.USER_INVITED_SUCCESSFULLY;
-import static io.harness.ng.core.invites.dto.InviteOperationResponse.USER_INVITE_NOT_REQUIRED;
+import static io.harness.ng.core.invites.dto.InviteOperationResponse.*;
```
Comment (pbarapatre10) [inline 120-ng-manager/src/test/java/io/harness/ng/core/invites/api/InviteServiceImplTest.java:15]: Remove the * import, it will fail in checkstyle

**Comment by pbarapatre10** [comment]
Please test for User explicit manual invite and SAML auto invite with/without 2FA, thanks

File: 120-ng-manager/src/test/java/io/harness/ng/core/invites/api/InviteServiceImplTest.java (hunk 1 above)
Comment (adiyaar24) [reply 120-ng-manager/src/test/java/io/harness/ng/core/invites/api/InviteServiceImplTest.java:15]: Ack , Intellij does this will update it

**Comment by adiyaar24** [comment]
> Please test for User explicit manual invite and SAML auto invite with/without 2FA, thanks

Check all the cases , works flawlessly

**Comment by adiyaar24** [comment]
trigger uts

**Comment by adiyaar24** [comment]
trigger uts

**Comment by adiyaar24** [comment]
trigger uts

**Review by pbarapatre10 — APPROVED** [review]

**Comment by adiyaar24** [comment]
trigger ut5, ut6, ut7, ut8, ut9

**Comment by adiyaar24** [comment]
trigger ut5

**Pull request merged**